sys-apps/archspec/archspec-0.1.3.ebuild

```diff
@@ -4,23 +4,26 @@
 EAPI=7
 
 PYTHON_COMPAT=( python3_{6..10} )
-
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
 inherit distutils-r1
 
-#DISTUTILS_USE_SETUPTOOLS=pyproject.toml
-
 DESCRIPTION="A library to query system architecture"
 HOMEPAGE="https://github.com/archspec/archspec"
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Maybe add a comment on top to explain why you're pulling the sources from GitHub rather than PyPI, and to mention why you need to download `archspec-json` separately?

Reply:
Done!